_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -15,6 +15,12 @@ accessibility_requirements:
     failed: not satisfied
     passed: further testing needed
     inapplicable: further testing needed
+  using-aria:fourth_rule_of_aria_use:
```

Comment:
I'm not sure what these keys are translated to behind the scenes, but I'm guessing _something_ needs awareness of `using-aria` for this to work. @jkodu?

Reply:
Yeah no, this isn't supported yet, but we should add it in. It's a good idea. This is not an ID in the document though. This should probably be `using-aria:4thrule`

Reply:
Yes, @jkodu helped me spotting the code that handle these keys and I will make a companion PR in the web repo.

Reply:
PR for supporting these keys in the website code: https://github.com/act-rules/act-rules-web/pull/56